src/inverseRobot.js

```diff
@@ -7,7 +7,16 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
```

Comment:
Consider using a more descriptive name for the 'newRobot' variable, such as 'invertedRobot'.